src/main/java/thePackmaster/cards/grandopeningpack/PlannedDefense.java

```diff
@@ -37,6 +39,21 @@ public void use(AbstractPlayer p, AbstractMonster m) {
         }
     }
 
+    @Override
+    public void triggerOnGlowCheck() {
+        boolean innatePlayed = false;
+        this.glowColor = AbstractCard.BLUE_BORDER_GLOW_COLOR.cpy();
```

Comment:
Please don't do this, you're creating a new color every time even when it might be overriden by gold. Instead just add an else to the if at the end and then set the glowColor to blue there

Reply:
I copied this from Ripped Pecs, which also first makes it glow blue and then gold, i can change that too if you want, or is there some difference i am not seeing?